app/atoms/ui/_general.scss

```diff
@@ -0,0 +1,57 @@
+/* margin values made variables */
+$margin_0: 0;
+$margin_0_auto: 0 auto;
+
+@mixin padding_section {
+  padding: 6em 0;
+}
+
+@mixin padding_section_reduced {
+  padding: 3em 0;
+}
+
+// put a hard limit on how narrow the screen can be: 320px
+body {
+  min-width: 320px;
+}
+
+// container class, for all sections that need to fill the screen with color beyond their limit
+.container {
+  @include outer-container;
+
+  @include media($tablet) {
+    width: 90%;
+    max-width: $max-width;
+    left: 0;
+    right: 0;
+    margin-left: auto;
```

Comment:
@adamf321 this has position static , no need left , and right 0 

I think is better remove all the attr and use only padding-left and right for the container in small devices